starters/cms/src/data.ts

```diff
@@ -0,0 +1,182 @@
+import type { CollectionItemData, ManagedCollection, ManagedCollectionField } from "framer-plugin"
+
+import { framer } from "framer-plugin"
+import { PLUGIN_KEYS } from "./constants"
+
+export type ManagedCollectionFieldType = ManagedCollectionField["type"]
+
+export interface DataSource {
+    id: string
+    fields: readonly ManagedCollectionField[]
+    items: Record<string, unknown>[]
+}
+
+export function getDataSourceIds(): readonly string[] {
+    return ["articles", "categories"]
+}
+
+export async function getDataSource(dataSourceId: string, abortSignal?: AbortSignal): Promise<DataSource> {
+    const dataSource: {
+        id: string
+        fields: {
+            name: string
+            type: string
+        }[]
+        items: Record<string, unknown>[]
+    } = await fetch(`/datasources/${dataSourceId}.json`, { signal: abortSignal }).then(response => response.json())
+
+    const fields: ManagedCollectionField[] = []
+    for (const field of dataSource.fields) {
+        switch (field.type) {
+            case "html":
+                fields.push({
+                    id: field.name,
+                    name: field.name,
+                    type: "formattedText",
+                    userEditable: false,
+                })
+                break
```

Comment:
I think we can just remove this if its not being used. this is more of an example for the docs/readme than something that should be included in every file I think